sanic_jwt/utils.py

```diff
@@ -40,14 +40,15 @@ def build_claim_aud(attr, *args, **kwargs):
     return attr
 
 
+# To be depracated
 def load_settings(app, settings):
     for setting in dir(settings):
         if setting.isupper() and setting not in app.config:
             value = getattr(settings, setting)
             setattr(app.config, setting, value)
 
 
-async def call_maybe_coro(fn, *args, **kwargs):
+async def call(fn, *args, **kwargs):
```

Comment:
I have written the first version of this method with this name. The reason I renamed the function to `call_maybe_coro` was that `__call__` is somewhat related to classes and also `subprocess.call` is widely used - well, `call` for me reminds me of both, instantly, and I don't wanted to get someone confused because of its hybrid sync/async nature. Personally I have no problem with that, I was just thinking about semantics :smiley:

Reply:
fair enough. we can set it back.

Reply:
I see no problem with `call`, since it's an internal utility. It was more like a comment ... There's no need to rework that.